test/evergreen.yml

```diff
@@ -115,6 +115,16 @@ functions:
         rm -rf "wiredtiger"
         rm -rf "wiredtiger.tgz"
 
+  "test checkpoint":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/build_posix/test/checkpoint"
+      script: |
+        set -o errexit
+        set -o verbose
+        ./t -t ${fileType} -n 1000000 -k 5000000 -C cache_size=100MB 2>&1
```

Comment:
I think we should wrap this whole part in a variable
-t ${fileType} -n 1000000 -k 5000000 -C cache_size=100MB

Reply:
> I think we should wrap this whole part in a variable
> -t ${fileType} -n 1000000 -k 5000000 -C cache_size=100MB

Wrapped the whole parameters into a variable "$testParams"